### ISSUE TITLE
Missing meet

### ISSUE THREAD

**Issue by GuilhermeIsep** [sstangl/openpowerlifting#1264]
The 2018 Portugal National Championship meet is missing.
Federation: WPC 
Untested meet

A link with the meet results:
https://imgur.com/a/NxUsXN9

Or image if you prefer:
![wpc](https://user-images.githubusercontent.com/29103868/40419211-84b9e2ec-5e7c-11e8-95e9-27120c7a8137.jpg)

The spreadsheet is written in Portuguese so if you need any help contact me:
instagram:  aleluialifts
or guilherm.oliveira90@gmail.com

**Comment by sstangl** [comment]
The image is barely readable -- do you have the original document as a spreadsheet?

It's extremely difficult, time-consuming, and error-prone to enter in meet results from images, so we vastly prefer not doing that.

**Comment by GuilhermeIsep** [comment]
Hey @sstangl  I'll make an excel spreadsheet with the results. Will you post the meet results that way?

Thank you

**Comment by sstangl** [comment]
Yes, that would be very helpful. Please keep it in a similar format as the JPG's so that we can verify that they match and look for transcription errors. Thanks for offering to do that!

**Comment by GuilhermeIsep** [comment]
@sstangl  No problem. I'll post it over here in a comment tomorrow.

**Comment by GuilhermeIsep** [comment]
It's going to take me a bit longer.  I'll post the excel on Sunday. Sorry for the delay

**Comment by GuilhermeIsep** [comment]
Here are the results. I couldn't read the Equipped division results. Also I could only put the results of 2 lifters of Raw Classic because they're my friends and I asked them their results. But as far as the Raw Divison goes, everything is all right I think. 


Results part1:
![1](https://user-images.githubusercontent.com/29103868/40586960-07f2df96-61c1-11e8-93cc-48a38d578fcf.PNG)


Results part2:
![2](https://user-images.githubusercontent.com/29103868/40586962-0ccb13a8-61c1-11e8-9dfd-1104b38e4395.PNG)

Any further questions don't hesitate, I'm here to contribute!

Thanks.

**Comment by GuilhermeIsep** [comment]
@sstangl

**Comment by photon-rain** [comment]
@GuilhermeIsep Could you post the spreadsheet that you made? Otherwise we'll have to retype it again.

**Comment by GuilhermeIsep** [comment]
[WPC-PortugalNationals-2018.xlsx](https://github.com/sstangl/openpowerlifting/files/2071831/WPC-PortugalNationals-2018.xlsx)

Here!

**Comment by photon-rain** [comment]
Thanks! Sorry that we hadn't got back to you, we'll try to get this up this week.
